server/app/routes/api/v1/annotations/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 var router = require('express').Router();
-module.exports = router;
+var _ = require('lodash');
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require('../../../utils');
+const {ensureAuthenticated, ensureIsAdmin, Credentials, respondWith404, _err, db} = utils;
+const Annotation = db().models.annotation;
+const Resource = Annotation;
```

Comment:
What is the point of having this named twice differently (i.e. Annotation vs Resource)?